src/content/docs/apm/agents/net-agent/net-agent-api/net-agent-api.mdx

```diff
@@ -724,95 +702,32 @@ The following list contains the different calls you can make with the API, inclu
               </td>
 
               <td>
-                Required. Container where trace context headers are inserted..
+                Required. A key/value pair store where distributed tracing data was inserted by the calling service. This needs to exist in whatever message object is being passed from the calling service to the called service, via whatever transport is being used.  For example, for Azure Service Bus messages, the `ServiceBusReceivedMessage` type has an [`ApplicationProperties` property](https://learn.microsoft.com/en-us/dotnet/api/azure.messaging.servicebus.servicebusmessage.applicationproperties?view=azure-dotnet#azure-messaging-servicebus-servicebusmessage-applicationproperties) which is an `IDictionary<string,object>`.
```

Comment:
Extra space before "for example"

Clean up this wordy sentence: "This needs to exist in whatever message object is being passed from the calling service to the called service, via whatever transport is being used."